v1/translate.yaml

```diff
@@ -0,0 +1,254 @@
+info:
+  version: '1.0.0'
+  title: Content Translation API
+  description: Content Translation API
+  termsofservice: https://github.com/wikimedia/restbase
+  license:
+    name: Apache licence, v2
+    url: https://www.apache.org/licenses/LICENSE-2.0
+paths:
+  /machine/{from}/{to}{/provider}:
```

Comment:
Even if we changed the top-level route to `translation`, `translation/translate` is still redundant. `mt` is too CXServer-specific (and cryptic from the user's perspective), while `machine-translation` is too long IMHO. I don't understand the problem you have with `machine` - `translate/machine/` describes correctly what is about to happen: a text will be machine-translated.